modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/KeyValueGetPlan.java

```diff
@@ -130,7 +130,7 @@ public IgniteKeyValueGet lookupNode() {
     @Override
     public <RowT> AsyncCursor<InternalSqlRow> execute(
             ExecutionContext<RowT> ctx,
-            @Nullable InternalTransaction tx,
+            InternalTransaction tx,
```

Comment:
seems additional assert will be useful ? 
assert tx.implicit();

Reply:
Key-value plan can be executed within an explicit (user-managed) transaction, in which case this assertion will fail